src/Proxy/ContainerInterfaceProxy.php

```diff
@@ -0,0 +1,275 @@
+<?php
+
+namespace Yiisoft\Yii\Debug\Proxy;
+
+use Psr\Container\ContainerInterface;
+use Psr\Container\ContainerExceptionInterface;
+use Psr\EventDispatcher\EventDispatcherInterface;
+use Yiisoft\Di\ContainerProxyInterface;
+use Yiisoft\Proxy\ProxyManager;
+use Yiisoft\Yii\Debug\Collector\CommonServiceCollectorInterface;
+use Yiisoft\Yii\Debug\Event\ProxyMethodCallEvent;
+
+class ContainerInterfaceProxy implements ContainerProxyInterface
+{
+    const LOG_ARGUMENTS = 1 << 0;
```

Comment:
Use already compiled numbers for constants values.

Reply:
I'm too lazy for that:)